### PR TITLE
feat: show max count of sessions that users have to an admin

### PR DESCRIPTION
**Pull request by Tymek**
Add info about large number of parallel sessions per user.

### CODE REVIEW

File: frontend/src/component/banners/internalBanners/SecurityBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        !isAdmin ||
+        !showUserDeviceCount ||
+        !uiConfig.maxSessionsCount ||
+        uiConfig.maxSessionsCount < 5
+    ) {
+        return null;
+    }
+
+    const banner = {
+        message: `Potencial security issue: there are ${uiConfig.maxSessionsCount} parallel sessions for a single user account.`,
```
Comment (kwasniew) [inline frontend/src/component/banners/internalBanners/SecurityBanner.tsx:23]: potential

File: src/test/e2e/api/admin/config.e2e.test.ts [hunk 2]
```diff
@@ -98,3 +99,73 @@ test('sets ui config with frontendSettings', async () => {
...
+    test('should return max sessions count', async () => {
+        const { body: noLoggedInUsers } = await app.request
+            .get(`/api/admin/ui-config`)
+            .expect('Content-Type', /json/)
+            .expect(200);
+
+        expect(noLoggedInUsers.maxSessionsCount).toEqual(0);
+    });
+
+    test('should return max sessions count', async () => {
```
Comment (kwasniew) [inline src/test/e2e/api/admin/config.e2e.test.ts:130]: should be different name

File: src/test/e2e/api/admin/config.e2e.test.ts [hunk 3]
```diff
@@ -98,3 +99,73 @@ test('sets ui config with frontendSettings', async () => {
...
+        }
+
+        const { body: withSessions } = await app.request
+            .get(`/api/admin/ui-config`)
+            .expect('Content-Type', /json/)
+            .expect(200);
+
+        expect(withSessions.maxSessionsCount).toEqual(5);
+
+        await app.services.sessionService.deleteSessionsForUser(user.id);
```
Comment (kwasniew) [inline src/test/e2e/api/admin/config.e2e.test.ts:169]: still needed?

**Review by kwasniew — APPROVED** [review]

File: frontend/src/component/banners/internalBanners/SecurityBanner.tsx (hunk 1 above)
Comment (Tymek) [inline frontend/src/component/banners/internalBanners/SecurityBanner.tsx:23]:
```suggestion
        message: `Potential security issue: there are ${uiConfig.maxSessionsCount} parallel sessions for a single user account.`,
```

**Pull request merged**